src/Rocketlet.ts

```diff
@@ -78,12 +82,18 @@ export abstract class Rocketlet {
      *
      * @return whether the Rocketlet should be enabled or not
      */
-    public abstract onEnable(): boolean;
+    public abstract onEnable(environment: IEnvironmentRead, configurationModify: object): boolean; //todo: Config modify. This should actually be an implementation of configModify which ensures that only own configurations are being modified
```

Comment:
that's how it thought you intended the methods to be used - and I like it. So: yes.
What's nice about it is that the really static things like adding configuration are done in `initialize` - without knowing the environment. 
In `enable/disable`, they basically know the world of configuration and can enable appropriately (e. g. modify default settings from `initialize` based on environment variables)